e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -17,6 +17,7 @@ import {
   filterWidget,
   getNotebookStep,
   modal,
+  notebookButton,
```

Comment:
1. New > Question > Orders > Save > Visualize

:x: The "notebook" button is active
:question: the results/visualization button is not shown

![image](https://github.com/user-attachments/assets/d5b8747f-6f7b-44ab-85d7-edea20ac70cc)

Reply:
that's correct. same behavior you'll see on stats. but active state is probably not something we'd like to see, unfortunately segment control marks the item active even if I don't pass any value to it.

I'll double check with Alessio

Reply:
Is this a visualization whose chart type is table? Is so, the 3 segments should be shown and this should be the viz, not the editor. Let me know if I'm interpreting this wrong.

Reply:
yes, it's a table chart type and in master we do not show controls to switch between viz and tabular data for a table, that's why it's implemented like this

https://github.com/user-attachments/assets/a55666d8-013a-4a0a-bca6-615f744eef14

I don't mind if we show all 3 segments anyway

Reply:
I see. Not switching between results table and viz table can make sense, but let's at least show two segments for the editor + table.